modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFolderResourceImpl.java

```diff
@@ -27,4 +41,161 @@
 	scope = ServiceScope.PROTOTYPE, service = ObjectFolderResource.class
 )
 public class ObjectFolderResourceImpl extends BaseObjectFolderResourceImpl {
+
+	@Override
+	public void deleteObjectFolder(Long objectFolderId) throws Exception {
+		_objectFolderService.deleteObjectFolder(objectFolderId);
+	}
+
+	@Override
+	public ObjectFolder getObjectFolder(Long objectFolderId) throws Exception {
+		return _toObjectFolder(
+			_objectFolderService.getObjectFolder(objectFolderId));
+	}
+
+	@Override
+	public ObjectFolder getObjectFolderByExternalReferenceCode(
+			String externalReferenceCode)
+		throws Exception {
+
+		com.liferay.object.model.ObjectFolder objectFolder =
+			_objectFolderLocalService.getObjectFolderByExternalReferenceCode(
+				externalReferenceCode, contextCompany.getCompanyId());
+
+		return getObjectFolder(objectFolder.getObjectFolderId());
+	}
```

Comment:
Hey @marcelabc 

In the `ObjectDefinitionResourceImpl` we use the `_objectDefinitionService.fetchObjectDefinitionByExternalReferenceCode` because this method is used in 3 places. I didn't see the necessity to create a `_objectFolderService.getObjectFolderByExternalReferenceCode` because we have one method by ERC. Do you understand?